scripts/create_new_jsons.py

```diff
@@ -0,0 +1,271 @@
+"""create_new_jsons
+
+This script is used to create new json entries for new psu versions.
+This script will also update json indexes for the new psu versions.
+
+Note: YOU WILL STILL NEED TO:
+    1. Run sign.sh to sign the new psu version jsons
+    2. Ensure that the filenames for the `aqavit_results_link` enties are correct
+
+Requirements:
+    1. python 3.8 and higher
+
+Usage:
+  create_new_jsons --folder=<str> --new_version=<str> --psu_tag=<str>
+
+Options:
+    --folder=<str>          Folder containing the json files
+    --new_version=<str>     New version to be added
+    --psu_tag=<str>         Tag for the new version
+    --help                  Show this help message
+Example:
```

Comment:
Nit: Need the example when we have `Usage`?

Reply:
@joe-braley, my documentation style is to be as explicit as possible, which does include some redundancy. I like to show people an explicit usage case from the past, so it can be more clear as to what values they need to input in the future :)